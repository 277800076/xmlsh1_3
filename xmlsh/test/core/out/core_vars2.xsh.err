[E:/Work/DEI/github/xmlsh_r1/xmlsh/test/core/core_vars2.xsh line: 37]
Exception running: echo
org.xmlsh.core.CoreException: Invalid blank name for output variable
[E:/Work/DEI/github/xmlsh_r1/xmlsh/test/core/core_vars2.xsh line: 39]
Exception running: echo
org.xmlsh.core.CoreException: Invalid blank name for output variable
