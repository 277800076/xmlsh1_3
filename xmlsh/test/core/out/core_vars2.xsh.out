A[2] is b
